src/test/resources/write-document.xml

```diff
@@ -129,4 +129,13 @@
                                restTransformParameters="hello;key;world;value" restTransformParametersDelimiter=";"/>
     <marklogic:read-document config-ref="config" uri="/writeDocumentWithTransform"/>
   </flow>
+
+  <flow name="writeDocumentWithArrayInput">
+    <set-variable variableName="uris" value="#[['/metadataSamples/text/hello.text','/metadataSamples/json/hello.json']]"/>
+    <marklogic:read-array config-ref="config" uris="#[vars.uris]"/>
+    <marklogic:write-multiple-documents config-ref="config" format="TEXT" collections="test-data"
```

Comment:
We don't want to use `test-data` for any documents written by tests - that prevents them from being deleted when the next test runs. `test-data` is only for documents loaded by the test-app that are not meant to be updated or deleted.